Plugins/GitUIPluginInterfaces/ObjectId.cs

```diff
@@ -320,27 +301,47 @@ public unsafe string ToShortString(int length = 8)
             char* buffer = stackalloc char[Sha1CharCount];
             char* p = buffer;
 
-            Write(_i1);
-            Write(_i2);
-            Write(_i3);
-            Write(_i4);
-            Write(_i5);
+            Write64(_i1);
+            Write64(_i2);
+            Write32(_i3);
 
             return new string(buffer, 0, length);
 
-            void Write(uint i)
+            void Write64(ulong i)
             {
```

Comment:
❔
```suggestion
            void Write64(ulong i)
            {
                Write32(i >> 32);
                Write32(i & 0xFFFF_FFFF);
            }
```

Reply:
As #11222 was accepted, I will remove this code